scripts/python/tao_build.py

```diff
@@ -495,7 +495,9 @@ def bazel_build(target, flag=""):
 
         bazel_build(TARGET_TAO_COMPILER_MAIN, flag=flag)
         bazel_build(TARGET_DISC_OPT, flag=flag)
-        bazel_build(TARGET_DISC_REPLAY, flag=flag)
+        # TODO:(fl237079) Support disc_replay for rocm version 
+        if not args.rocm and not args.dcu:
+            bazel_build(TARGET_DISC_REPLAY, flag=flag)
```

Comment:
What's the problem when building replay tool for rocm?

Reply:
There are some GPU related library dependency problems.